circuits-circom/regexes/venmo_receive_id.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parts": [
+    {
+      "is_public": false,
+      "regex_def": "<!-- actor name -->\r\n"
+    },
+    {
+      "is_public": false,
+      "regex_def": "                <a style=3D\"color:#0074DE; text-decoration:none\" href=3D\"ht=\r\n"
+    },
+    {
+      "is_public": false,
+      "regex_def": "tps://venmo.com/code\\?user_id=3D"
+    },
+    {
+      "is_public": true,
+      "regex_def": "(0|1|2|3|4|5|6|7|8|9)+"
+    },
+    {
+      "is_public": false,
+      "regex_def": "&actor_id=3D"
+    },
+    {
+      "is_public": true,
+      "regex_def": "(0|1|2|3|4|5|6|7|8|9|\r|\n|=)+"
+    },
+    {
+      "is_public": false,
+      "regex_def": "\">\r\n"
+    }
```

Comment:
21 + 79 + 31 + 12 + 4 = 147. In total, we have only 147 static characters. I suggest we increase it such that we are well above the venmo custom message length.